server/tsconfig.json

```diff
@@ -1,26 +1,24 @@
 {
+  "extends": "@tsconfig/node16",
   "compilerOptions": {
-    "target": "es6",
-    "module": "commonjs",
-    "lib": ["ES6", "ES2017", "ES2018", "ES2019", "ES2020"],
-    "sourceMap": false,
-    "outDir": "./dist",
-    "rootDir": "src",
     "moduleResolution": "node",
-    "removeComments": true,
-    "noImplicitAny": true,
-    "strictNullChecks": true,
```

Comment:
Wouldn't it be useful to have `strictNullChecks`? So we don't call methods on something that's potentially undefined?